code/modules/mob/living/simple_animal/hostile/human/_humanoid.dm

```diff
@@ -62,3 +66,22 @@
 
 	qdel(src)
 	return
+
+/mob/living/simple_animal/hostile/humanoid/Shoot()
+	if(needs_to_reload)
```

Comment:
Inverse this if/else chain like so:

```dm
if (!needs_to_reload)
    ..()
    return

if (bullets_remaining > 0)
    bullets_remaining--
    ..()
    return

...
```

Basically, if no code executes after a branch you make it return instead of using an `else`, and use the shorter one for readability.